File: http patch/Scala/fundservice/fund/src/test/scala/PatchFundFixture.scala
import fund.DomainModel.Fund
import fund.common.Pipeline.Result

object PatchFundFixture {
  val HEDGE_FUND_FUND_TYPE = "Hedge Fund"
  val DRAWDOWN_VEHICLE_FUND_TYPE = "DrawDown Vehicle"
  val FUND_NAME = "Cardano Fund A"
  val FUND_ID = 1

  private var fundInDummyDb = Fund(1, FUND_NAME, HEDGE_FUND_FUND_TYPE)

  def getFund(fundId: Int): Fund = {
    fundInDummyDb
  }

  def writeToDb(updatedFund: Fund) = {
    fundInDummyDb = updatedFund
  }

  def updateFund(editedFund: Fund): Result[Fund] = {
    val existing = getFund(editedFund.fundId)
    val updated = Result(data = Fund(existing.fundId, editedFund.fundName, editedFund.fundType))

    writeToDb(updated.data)

    updated
  }

}

File: http patch/C#/FundService/FundService.Tests/PatchFundFixture.cs
using FundService.Common;

namespace FundService.Tests
{
    internal static class PatchFundFixture
    {
        internal const int FUND_ID_STUB = 1;
        internal const string FUND_NAME_STUB = "JFK Diversity";
        internal const string FUND_TYPE_STUB = "Hedge Fund";

        private static Fund dbFund = new Fund(FUND_ID_STUB, FUND_NAME_STUB, FUND_TYPE_STUB);

        internal static Fund GetFund(int id) {
            return dbFund.Id == id ? dbFund : null;
        }

        internal static Result<Fund> UpdateFund(Fund fund) {
            dbFund = new Fund(fund.Id, fund.FundName, fund.FundType);
            return new Result<Fund>(data: dbFund);
        }
    }
}
